### DISCUSSION TITLE
서버 이전

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#938] [이야기해봐요]
.

**Comment by jminkkk** [comment]
# Prod DB 이전

1. dump 따기 (Prod-a에서)
```
mysqldump --no-tablespaces --set-gtid-purged=OFF -h database-project.cluster-cqsc6pyqhwww.ap-northeast-2.rds.amazonaws.com -P 3306 -u codezap -p codezap > codezap_dump.sql
```

2. dump 파일 다운

```
exec mysql -u root -p < /codezap_dump.sql
```

3. 
```
```

**Comment by kyum-q** [comment]
SSL 인증서

**Reply by kyum-q** [reply]
WARN[0000] The "HOSTNAME" variable is not set. Defaulting to a blank string. 
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Requesting a certificate for *.code-zap.com
Client with the currently selected authenticator does not support any combination of challenges that will satisfy the CA. You may need to use an authenticator plugin that can do challenges over DNS.
Client with the currently selected authenticator does not support any combination of challenges that will satisfy the CA. You may need to use an authenticator plugin that can do challenges over DNS.
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /var/log/letsencrypt/letsencrypt.log or re-run Certbot with -v for more details.
ubuntu@ip-172-31-45-50:~/zap$

**Reply by kyum-q** [reply]
-
Press Enter to Continue

Certbot failed to authenticate some domains (authenticator: manual). The Certificate Authority reported these problems:
  Domain: code-zap.com
  Type:   unauthorized
  Detail: No TXT record found at _acme-challenge.code-zap.com

Hint: The Certificate Authority failed to verify the manually created DNS TXT records. Ensure that you created these in the correct location, or try waiting longer for DNS propagation on the next attempt.

Some challenges have failed.
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /var/log/letsencrypt/letsencrypt.log or re-run Certbot with -v for more details.]

**Reply by kyum-q** [reply]
Hint: The Certificate Authority failed to verify the manually created DNS TXT records. Ensure that you created these in the correct location, or try waiting longer for DNS propagation on the next attempt.

Some challenges have failed.
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /var/log/letsencrypt/letsencrypt.log or re-run Certbot with -v for more details.
